datafusion/functions/src/string/btrim.rs

```diff
@@ -124,6 +124,8 @@ fn get_btrim_doc() -> &'static Documentation {
 ```"#)
             .with_standard_argument("str", Some("String"))
             .with_argument("trim_str", "String expression to operate on. Can be a constant, column, or function, and any combination of operators. _Default is whitespace characters._")
+            .with_alternative_syntax("trim(BOTH trim_str FROM str)")
+            .with_alternative_syntax("trim(trim_str FROM str)")
```

Comment:
Does this deserve new SLT test cases?

Reply:
I grabbed the syntax from an slt test :)